tests/unit/AmalgamatingBusinesses.spec.ts

```diff
@@ -115,48 +114,53 @@ describe('Amalgamating Businesses - components and validity', () => {
-    { type: AmalgamationTypes.HORIZONTAL, haveHoldingBusiness: true, haveAmalgamatingBusinesses: true, allOk: false, expected: false },
-    { type: AmalgamationTypes.HORIZONTAL, haveHoldingBusiness: true, haveAmalgamatingBusinesses: true, allOk: true, expected: true }
+    { type: AmalgamationTypes.REGULAR, haveAmalgamatingBusinesses: false, allOk: false, expected: false },
+    { type: AmalgamationTypes.REGULAR, haveAmalgamatingBusinesses: false, allOk: true, expected: false },
+    { type: AmalgamationTypes.REGULAR, haveAmalgamatingBusinesses: true, allOk: false, expected: false },
+    { type: AmalgamationTypes.REGULAR, haveAmalgamatingBusinesses: true, allOk: true, expected: true },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: false, haveAmalgamatingBusinesses: false, allOk: false, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: false, haveAmalgamatingBusinesses: false, allOk: true, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: false, haveAmalgamatingBusinesses: true, allOk: false, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: false, haveAmalgamatingBusinesses: true, allOk: true, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: true, haveAmalgamatingBusinesses: false, allOk: false, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: true, haveAmalgamatingBusinesses: false, allOk: true, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: true, haveAmalgamatingBusinesses: true, allOk: false, expected: false },
+    { type: AmalgamationTypes.VERTICAL, haveHoldingBusiness: true, haveAmalgamatingBusinesses: true, allOk: true, expected: true }
   ]
 
   for (let i = 0; i < businessTableValidTests.length; i++) {
-    it.skip(`computes businessTableValid correctly - test ${i}`, () => {
+    it(`test ${i}`, () => {
+      store.stateModel.tombstone.filingType = FilingTypes.AMALGAMATION_APPLICATION
       store.stateModel.amalgamation.type = businessTableValidTests[i].type
 
-      // mount a new wrapper to mock computed property and set data
       // NB: use shallowMount so BusinessTable doesn't affect data
-      const wrapper2 = shallowMount(AmalgamatingBusinesses, {
+      const wrapper = shallowMount(AmalgamatingBusinesses, {
         computed: {
-          haveAmalgamatingBusinesses: () => businessTableValidTests[i].haveAmalgamatingBusinesses,
-          haveHoldingBusiness: () => businessTableValidTests[i].haveHoldingBusiness
+          haveAmalgamatingBusinesses: {
+            get (): boolean { return businessTableValidTests[i].haveAmalgamatingBusinesses }
+          },
+          haveHoldingBusiness: {
+            get (): boolean { return businessTableValidTests[i].haveHoldingBusiness }
+          }
         },
         data: () => ({ allOk: businessTableValidTests[i].allOk }),
```

Comment:
And this is how you override/initialize a local property (variable). Note that processing in the component could overwrite this, however. That's why I shallow-mounted this component.